ouroboros-consensus/CHANGELOG.md

```diff
@@ -2,6 +2,13 @@
 
 # Changelog entries
 
+<a id='changelog-0.6.1.0'></a>
+## 0.6.1.0 — 2023-05-19
```

Comment:
Damn, I forgot to include a "Breaking" fragment in #89 :facepalm: I.e. like this:

```
### Breaking

 - Renamed `TranslateForecast` to `CrossEraForecaster` and `translateLedgerView` to `crossEraForecast`.
```
and bump `ouroboros-consensus` to `0.7.0.0`.

These types/fields are not used anywhere outside of our packages AFAICT, but e.g. `ouroboros-consensus-cardano-0.5.1.0` would fail to build with `ouroboros-consensus-0.6.1.0` (if we would release it as we do here).